stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -1989,7 +1989,7 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     @inlinable // This is @inlinable as a convenience initializer.
     public init(contentsOf url: __shared URL, options: Data.ReadingOptions = []) throws {
         let d = try NSData(contentsOf: url, options: ReadingOptions(rawValue: options.rawValue))
-        self.init(bytes: d.bytes, count: d.length)
```

Comment:
Since this init method is inlinable, does this change mean anything one way or another after ABI stability? I assume just that new clients will get the new code and old will exist with the old code?

Reply:
Any call site which has had this inlined will not benefit from the change until recompilation, but any call site which has not (along with new clients regardless of inlining) will get the fix.

It is possible for call sites *within* a target to be inlined sometimes and not always, I believe, but since we're just working to avoid an additional copy instead of changing behavior, I'm less concerned about potential for inconsistency.